libs/error-stack/.justfile

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env just --justfile
+
+set fallback
+
+[private]
+@default:
+  just usage
+
+# TODO: add `--ignore-unknown-features` to `cargo hack` and pass `--workspace`
```

Comment:
What's the importance of this TODO? Is it tracked or handled in a future PR?

Reply:
It's an annoyance and unimportant until we implement something in `error-stack` macros. It was just ported over.